test/utils/tags.go

```diff
@@ -0,0 +1,104 @@
+package utils
+
+import (
+	"context"
+	"strings"
+
+	"github.com/aws/aws-sdk-go-v2/aws"
+	"github.com/aws/aws-sdk-go-v2/config"
+	"github.com/aws/aws-sdk-go-v2/service/kms"
+	"github.com/aws/aws-sdk-go-v2/service/resourcegroupstaggingapi"
+	tagging_types "github.com/aws/aws-sdk-go-v2/service/resourcegroupstaggingapi/types"
+	"github.com/aws/aws-sdk-go-v2/service/secretsmanager"
+)
+
+const maxPages = 10
+
+// GetTaggedResources -
+func GetTaggedResources(ctx context.Context, environmentId, assetId string) ([]string, error) {
+
+	cfg, err := config.LoadDefaultConfig(ctx)
+	if err != nil {
+		return nil, err
+	}
+
+	c := resourcegroupstaggingapi.NewFromConfig(cfg)
+
+	var arnSlice []string
+
+	params := &resourcegroupstaggingapi.GetResourcesInput{
+		TagFilters: []tagging_types.TagFilter{
+			{Key: aws.String("aptible_asset_id"), Values: []string{assetId}},
+			{Key: aws.String("aptible_environment_id"), Values: []string{environmentId}},
+		},
+	}
+
+	for i := 0; i < maxPages; i++ {
+		response, err := c.GetResources(ctx, params)
```

Comment:
i'd probably either do a logger statement if max pages is hit OR throw an error that pagination limit exceeded, not sure which so people know the results list is possibly truncated (though unlikely)